src/virtual.jl

```diff
@@ -311,6 +311,56 @@ function _get_integrated_master(b)
   return check
 end
 
+function get_cavity_frequency(ele::LineElement, key::Symbol)
+  c = ele.RFParams
+  if isnothing(c)
+    error("Unable to get $key of LineElement $(ele.name): No RFParams present")
```

Comment:
```suggestion
    return nothing
```

The new convention is that unpresent values return `nothing`